file/writable_file_writer.cc

```diff
@@ -187,7 +187,20 @@ IOStatus WritableFileWriter::Flush() {
     }
   }
 
-  s = writable_file_->Flush(IOOptions(), nullptr);
+  {
+    IOSTATS_TIMER_GUARD(write_nanos);
```

Comment:
Are we measuring write here?

Reply:
It seems I misunderstood the meaning of "write" here. Considering the entries in IOStatsContext (iostats_context.h), I may need a new entry for flush?

Reply:
We have not been measuring it, so I guess just removing the `IOSTATS` line here is good for this PR.

Reply:
Copy.